conf/resinhup.conf

```diff
@@ -53,7 +53,9 @@ to_keep_files:
 [fetcher]
```

Comment:
"The resinhup update package have changes the format" - this still throws me off